src/collective/behavior/amp/browser/__init__.py

```diff
@@ -114,6 +115,11 @@ def amp_analytics(self):
         amp_analytics = IAMPSettings.__identifier__ + '.amp_analytics'
         return api.portal.get_registry_record(amp_analytics)
 
+    @property
+    def text(self):
+        util = Html2Amp()
+        return util(self.context.text.output)
```

Comment:
you need to be sure that `self.context` has an attribute called `text` and that attribute is in fact a `RichTextValue` or this will break.